lib/sync/fetcher_test.go

```diff
@@ -82,3 +83,19 @@ func TestBlockFetcher(t *testing.T) {
 	require.Equal(t, bk.Hash, si.Block.Hash)
 	require.Equal(t, bk.TransactionsRoot, si.Block.TransactionsRoot)
 }
+
+func TestLargeFetch(t *testing.T) {
+	f := &BlockFetcher{}
+	f.logger = log
+
+	b := bytes.NewReader([]byte(largeData))
+
+	items, err := f.unmarshalResp(b)
+	require.NoError(t, err)
+	require.Equal(t, len(items), 2)
+
```

Comment:
Extra whiteline

Reply:
IMHO, we need to reduce tx payload.....

Reply:
You are right, but my comment is about the empty line before `}`